exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SQLExpressionBuilder.kt

```diff
@@ -960,6 +1013,18 @@ interface ISqlExpressionBuilder {
         return SingleValueInListOp(this, list.map { EntityIDFunctionProvider.createEntityID(it, idTable) }, isInList = false)
     }
 
+    /**
+     * Checks if this [EntityID] column is not equal to any element from [list].
+     *
+     * @sample org.jetbrains.exposed.sql.tests.shared.entities.CompositeIdTableEntityTest.testInListWithCompositeIdEntities
+     */
+    @Suppress("UNCHECKED_CAST")
+    @JvmName("notInListCompositeEntityIds")
+    infix fun <ID : EntityID<CompositeID>> Column<ID>.notInList(list: Iterable<CompositeID>): InListOrNotInListBaseOp<List<*>> {
```

Comment:
Idea says that there are no usages of that function.

Actually, if I comment this function, and the one above (`infix fun List<Column<*>>.notInList(list: Iterable<CompositeID>): InListOrNotInListBaseOp<List<*>>`) I get tests green (at least for H2 and Postgres), is it just a lack of tests yet?

Reply:
They are being tested in the test that was originally commented out before PR #2176 was merged.